### ISSUE TITLE
web-actions

### ISSUE THREAD

**Issue by KoenHolleboom** [mendix/docs#6135]
Please use the form below, leaving the prefilled data to help us. Thank you.

Page link: [web-actions](https://docs.mendix.com/appstore/modules/web-actions/)

Document link: [web-actions.md](https://github.com/mendix/docs/blob/development/content/en/docs/appstore/modules/web-actions.md)

## My Issue/Suggestion

It does not give any hands-on manual on how to use and/or set up any of the other features, like scroll-to or set-focus.
In other words: it is to much code-monkey focused and not low-code-business-engineer focused

**Comment by Luyao-Zhang-1** [comment]
@KoenHolleboom  Thank you for sharing your feedback with us. I will discuss with the owner of the module and get back to you on this. Since many people are on summer vacation, it might take a while before I can give you an update. But I will try to resolve soon.
